### PR TITLE
Feature/#508 change pipeline config documentation pictures

### PR DESCRIPTION
**Pull request by toan-quach**

### CODE REVIEW

File: docs/manuals/core/concepts/sequence.md [hunk 1]
```diff
@@ -8,9 +8,8 @@ preprocessing and preparing data, a sequence for computing a training model, and
 
     We have chosen to model only two sequences corresponding to a manufacturer having first to predict the sales
     forecast then, based on the sales forecast, plan its production in its plant.
```
Comment (erikfTW) [inline docs/manuals/core/concepts/sequence.md]:
```suggestion
    forecast, then, based on the sales forecast, plan its production in its plant.
```

File: docs/manuals/core/config/job-config.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ available in Taipy: the `standalone` mode and the `development` mode.
 With the _development_ mode, the jobs are synchronously executed one by one. The jobs are directly executed
```
Comment (erikfTW) [inline docs/manuals/core/config/job-config.md:7]:
```suggestion
With the _development_ mode, the jobs are synchronously executed individually. The jobs are directly executed
```

File: docs/manuals/core/config/job-config.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ available in Taipy: the `standalone` mode and the `development` mode.
...
 in the main thread at the submission. Note that with the _development_ mode, the submit method waits for the
 jobs to be finished before to return. Please refer to the
```
Comment (erikfTW) [inline docs/manuals/core/config/job-config.md:9]:
```suggestion
jobs to be finished before returning.
```

File: docs/manuals/core/config/job-config.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ available in Taipy: the `standalone` mode and the `development` mode.
...
-[submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-pipeline-or-task) section to
+[submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task) section to
 the see how to submit jobs.
```
Comment (erikfTW) [inline docs/manuals/core/config/job-config.md:11]:
```suggestion
For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).
```

File: docs/manuals/core/config/job-config.md [hunk 5]
```diff
@@ -56,7 +56,7 @@ At the submission, the job is queued. It is dequeued in a different thread and s
 executed.
 Note that with the _standalone_ mode, the submit method in not blocking and returns after the job is queued.
 It means the submit method can return before the job finishes or even before it is dequeued. Please refer to
```
Comment (erikfTW) [inline docs/manuals/core/config/job-config.md:58]:
```suggestion
It means the submit method can return before the job finishes or even before it is dequeued. 
```

File: docs/manuals/core/config/job-config.md [hunk 6]
```diff
@@ -56,7 +56,7 @@ At the submission, the job is queued. It is dequeued in a different thread and s
...
-the [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-pipeline-or-task) section
+the [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task) section
 to the see how to submit jobs.
```
Comment (erikfTW) [inline docs/manuals/core/config/job-config.md:60]:
```suggestion
For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).
```

**Review by erikfTW — CHANGES_REQUESTED** [review]

**Review by jrobinAV — APPROVED** [review]
Looks good to me. 
Thx.
When it s merged we can start discussing how to restructure the user man, @erikfTW. Probably after the getting started though.

File: docs/manuals/core/config/job-config.md (hunk 3 above)
Comment (erikfTW) [reply docs/manuals/core/config/job-config.md:9]: My suggestion is: 

For more information on how to submit jobs, refer to [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task).

Comment (toan-quach) [reply docs/manuals/core/config/job-config.md:9]: Haha yep I just saw it below! Sorry for the trouble! Have fixed and pushed the latest changes :D

Comment (erikfTW) [reply docs/manuals/core/config/job-config.md:9]: Ok cool, thanks! :)

**Review by erikfTW — APPROVED** [review]

**Pull request merged**